src/core/evaluator.js

```diff
@@ -4787,6 +4788,37 @@ class TranslatedFont {
       i++;
     }
   }
+
+  #guessType3FontBBox(operatorList) {
+    if (typeof PDFJSDev === "undefined" || PDFJSDev.test("TESTING")) {
+      assert(
+        operatorList.fnArray[0] === OPS.setCharWidth,
+        "Type3 glyph shall start with the d0 operator."
+      );
+    }
+
+    let i = 1;
+    const ii = operatorList.length;
+    while (i < ii) {
+      switch (operatorList.fnArray[i]) {
+        case OPS.constructPath:
+          const minMax = operatorList.argsArray[i][2];
```

Comment:
the minMax doesn't take into account the Bezier curves we could have, right ?

Reply:
It does not, as mentioned in https://github.com/mozilla/pdf.js/blob/1bc98dfbd92a832d34ff8151c529368ac2f8c8c6/src/core/evaluator.js#L1431-L1435

However, this patch is for a particular (and hopefully) rare case of corrupt Type3 fonts and as-is it's already enough to fix the issue so I'm thinking it's fine.